pyvista/core/dataset.py

```diff
@@ -776,6 +776,39 @@ def active_scalars(self) -> Optional[pyvista_ndarray]:
                 return None
         return None
 
+    @property
+    def active_normals(self) -> Optional[pyvista_ndarray]:
```

Comment:
I think we should also define `active_normals_name` with a setter. And I see that `active_t_coords` allows setting, which I think we forbid for scalars... I guess this could use some consistency. New issue?

Reply:
Yes, the scope of this PR is growing...